### PR TITLE
Remove files no longer in use

### PR DESCRIPTION
**Pull request by webpro**
Changelogs are now maintained in Markdown files, the JSON files are no longer updated.

### CODE REVIEW

File: .changeset/ninety-ants-grow.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+---
+---
```
Comment (Takadenoshi) [inline .changeset/ninety-ants-grow.md:2]: ?

**Review by Takadenoshi — APPROVED** [review]

File: .changeset/ninety-ants-grow.md (hunk 1 above)
Comment (EnoF) [reply .changeset/ninety-ants-grow.md:2]: :ant:

**Review by realdreamer — APPROVED** [review]

**Review by EnoF — APPROVED** [review]

File: .changeset/ninety-ants-grow.md (hunk 1 above)
Comment (webpro) [reply .changeset/ninety-ants-grow.md:2]: We now have a system in place where we have to add a changelog in each PR. This syntax is not pretty, but it means a no bump (or "none") is necessary for any package. Generated using `pnpm changeset --empty`.

Comment (webpro) [reply .changeset/ninety-ants-grow.md:2]: It's also a free Discovery Channel episode.

Comment (webpro) [reply .changeset/ninety-ants-grow.md:2]: > 🐜🐜🐜🐜🐜🐜🐜🐜🐜🐜🐜🐜🐜🐜🐜🐜🐜🐜🐜🐜🐜🐜🐜🐜🐜🐜🐜🐜🐜🐜🐜🐜🐜🐜🐜🐜🐜🐜🐜🐜🐜🐜🐜🐜🐜🐜🐜🐜🐜🐜🐜🐜🐜🐜🐜🐜🐜🐜🐜🐜🐜🐜🐜🐜🐜🐜🐜🐜🐜🐜🐜🐜🐜🐜🐜🐜🐜🐜🐜🐜🐜🐜🐜🐜🐜🐜🐜🐜🐜🐜

I love that there are actually 90 ants. And that I counted them.

**Pull request merged**